app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
+
+    filename = argv[index]
+
+    if "-d" in argv:
+        index2 = 0
+
+        for _ in range(len(argv) - 1):
+            index2 += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
+
+        directories = argv[index2:index - 1]
+else:
+    filename = "file.txt"
+
+    if "-d" in argv:
+        index = 0
+
+        for _ in range(len(argv) - 1):
+            index += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
+
+        directories = argv[index:len(argv)]
+
+file_data = [datetime.today().strftime('%y:%d:%m %H-%M-%S'), "\n"]
+
+while True:
+    line = input("Enter content line: ")
+    if line == "exit":
+        break
+    file_data.extend([line])
+
+for directory in directories:
+    try:
+        mkdir(directory)
+    except Exception:
+        pass
+    chdir(directory)
+
+with open(filename, "w") as f:
```

Comment:
Instead of the chdir method, you can use the full path to the file. Also you have to use "a" - Append mode to add new lines instead of overwriting the file in "w" - Write mode. Because as described in the task, if the file already exists, it should add the content below.
```suggestion
with open(path.join(*directories, filename), "a") as f:
```